### DISCUSSION TITLE
Grammer correction issue

### DISCUSSION THREAD

**Discussion by Mansi9991** [openai/whisper#1275] [Q&A]
I am transcribing an audio file using the ```small``` model.. Everything is perfect while transcribing the audio..
But my question is, The ' WHISPER' model does auto-correction in grammer..

Such my audio file contains :
```"In this course, will we...."```

But it returns:
```"In this course. we will..."```

The problem is :
It corrects the ```grammer mistakes``` while transcribing audio file

Here I am attaching the link to the audio file::
```'https://drive.google.com/file/d/1lNjjKvZxJ8G_pH7i19oMhxLf2Oj5loiE/view?usp=share_link'```

**Comment by phineas-pta** [comment]
with whisper the result is "we will", with wav2vec2 it's "will we", maybe the audio too noisy to decide

but if i read the whole transcript, "we will" is more gramatically correct

**Reply by Mansi9991** [reply]
Yes it is correct gramatically but audio file contains 'will we' so the result should be 'will we' but it returns 'we will'

**Reply by phineas-pta** [reply]
how could you be sure it's "will we" ? did you test with other models ? for me the audio is too noisy to hear properly

**Reply by Mansi9991** [reply]
Yes.... I have tried with other models also and also done after denoising the audio file but still no change...It gives 'will we' after I change the value of 'compression_ration_threshold' to '1.2' from '2.4' but it affects the other transcription in the audio file but it is still not every time I get 'will we' after changing the value

**Reply by phineas-pta** [reply]
sorry i can't help, some models i tried (paddlespeech, nemo stt_en_conformer_transducer_xlarge) return another text: "will be [exploring]", i think it should be audio quality problem, not sure of auto-correction
